README.md

```diff
@@ -70,6 +70,27 @@ module mongodb_backup_advanced {
 }
 
 ```
+
```

Comment:
have some minimal example by using secret manager,


not sure the current added lines are useful,
please describe if there is something special with is data and variable lines having in this README.md file ?

Reply:
we still need this example how can this module be used with secret manager variables set